packages/components/src/components/modal/modal.jsx

```diff
@@ -50,13 +62,16 @@ class ModalElement extends React.PureComponent {
                         {this.props.header}
                     </div>
                     }
-                    <div onClick={this.props.toggleModal} className={classNames('dc-modal-header__close', this.props.className && `dc-modal-header__close--${this.props.className}`)} >
-                        <IconClose />
+                    <div
+                        onClick={this.props.toggleModal}
+                        className='dc-modal-header__close'
+                    >
+                        {this.props.has_close_icon && <IconClose />}
```

Comment:
should `has_close_icon` checks whether the parent `<div > ` should be rendered instead?
```
{ this.props.has_close_icon && 
   <div ...>
        <IconClose />
   </div> 
}
```

Reply:
Yeah, some of the modals don't need the close icon, but the container is there and adds visual artifacts.